examples/04-rejected-by-else.lc

```diff
@@ -0,0 +1,7 @@
+let id   = \x -> x
```

Comment:
File name should be changed to `04-rejected-by-elsa.lc`, `else` is a typo I think.